language/stdlib/transaction_scripts/remove_validator_and_reconfigure_admin.move

```diff
@@ -0,0 +1,21 @@
+script {
+    use 0x1::LibraSystem;
+    use 0x1::SlidingNonce;
+    use 0x1::ValidatorConfig;
+
+    /// Removes a validator from the validator set.
+    /// Fails if the validator_address is not in the validator set.
+    /// Emits a NewEpochEvent.
+    /// Can only be called as an administrative script to trigger waypoints.
+    fun remove_validator_and_reconfigure_admin(
+        _: &signer,
```

Comment:
why isn't LR the first param here?

Reply:
Both of these signers are the LR account: in the case that the admin script is sent as a waypoint this is the script that is used. There is also a non-admin version of this script as well that only takes the single `lr_account` signer (`remove_validator_and_reconfigure`).

Reply:
This was cleaned up by the single `execute_as` above?

Reply:
yep! This is because we were no longer passing in the doubled-up "execute_as" sender through (since this transaction was being sent by the VM for waypoints).